src/Tribe/Admin/Notice/Install_Event_Tickets.php

```diff
@@ -0,0 +1,436 @@
+<?php
+/**
+ * Install_Event_Tickets notice.
+ * Install and/or activate Event Tickets when it is not active.
+ */
+
+namespace Tribe\Events\Admin\Notice;
+
+use WP_Upgrader;
+use WP_Ajax_Upgrader_Skin;
+use Plugin_Upgrader;
+use Tribe__Main;
+use Tribe__Admin__Notices;
+use Tribe__Template;
+
+/**
+
+ */
+class Install_Event_Tickets {
+
+	/**
+	 * Stores the plugin slug.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected static $plugin_slug = 'event-tickets';
+
+	/**
+	 * Stores the assets group ID for the notice.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected static $assets_group = 'tribe-events-admin-notice-install-event-tickets';
+
+	/**
+	 * Stores the instance of the notice template.
+	 *
+	 * @since TBD
+	 *
+	 * @var Tribe__Template
+	 */
+	protected $template;
+
+	/**
+	 * Register update notices.
+	 *
+	 * @since TBD
+	 */
+	public function hook() {
+		if ( ! is_admin() || ! current_user_can( 'install_plugins' ) ) {
+			return;
+		}
+
+		add_action( 'wp_ajax_notice_install_event_tickets', [ $this, 'ajax_handle_notice_install_event_tickets' ] );
+
+		$this->assets();
+		$this->notice_install();
+		$this->notice_activate();
+	}
+
+	/**
+	 * Register `Install` notice assets.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function assets() {
+		$plugin = tribe( 'tec.main' );
+
+		tribe_asset(
+			$plugin,
+			'tribe-events-admin-notice-install-event-tickets-js',
+			'admin/notice-install-event-tickets.js',
+			[
+				'jquery',
+				'tribe-common',
+			],
+			null,
+			[
+				'localize' => [
+					[
+						'name' => 'TribeEventsAdminNoticeInstall',
+						'data' => [ 'ajaxurl' => admin_url( 'admin-ajax.php', ( is_ssl() ? 'https' : 'http' ) ), ]
+					],
+				],
+				'groups'   => [
+					self::$assets_group,
+				],
+			]
+		);
+
+		tribe_asset(
+			$plugin,
+			'tribe-events-admin-notice-install-event-tickets-css',
+			'admin/notice-install-event-tickets.css',
+			[ 'tec-variables-full' ],
+			[
+				'admin_enqueue_scripts',
+				'wp_enqueue_scripts',
+			],
+			[
+				'groups' => self::$assets_group,
+			]
+		);
+	}
+
+	/**
+	 * Get the plugin path for `Event Tickets`, by default.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $slug The plugin slug.
+	 *
+	 * @return string $path The plugin path.
+	 */
+	protected function get_plugin_path( $slug = '' ): string {
+		if ( empty( $slug ) ) {
+			$slug = self::$plugin_slug;
+		}
+
+		return $slug . '/' . $slug . '.php';
+	}
+
+	/**
+	 * Checks if `Event Tickets` is installed.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $slug The plugin slug.
+	 *
+	 * @return boolean True if active
+	 */
+	public function is_installed( $slug = '' ): bool {
+		$installed_plugins = get_plugins();
+
+		return array_key_exists( $this->get_plugin_path( $slug ), $installed_plugins ) || in_array( $this->get_plugin_path( $slug ), $installed_plugins, true );
```

Comment:
Checking for the whole path here leads to false negatives in some cases, plugins can be installed in any path. We should look for the `.php` file alone, which is a lot more unlikely to change, the folder can be changed easily.

Reply:
[See comment](https://github.com/the-events-calendar/the-events-calendar/pull/4110#discussion_r1085156434).